Rakefile

```diff
@@ -33,14 +33,12 @@ if cucumber?
   Cucumber::Rake::Task.new(:features) do |t|
     opts = "features --format junit -o #{CUKE_RESULTS} --format pretty -x"
     opts += " --tags #{ENV['TAGS']}" if ENV['TAGS']
-    opts += " --tags ~@skip"
     t.cucumber_opts = opts
     t.fork = false
   end
 
   desc 'Run features tagged as work-in-progress (@wip)'
   Cucumber::Rake::Task.new('features:wip') do |t|
-    tag_opts = ' --tags ~@pending'
```

Comment:
Removed this one because it is meaningless (overridden next line)
Also, @pending tag does not exist across .feature files.